### ISSUE TITLE
System Identification Guidelines

### ISSUE THREAD

**Issue by QiyangYan** [google-deepmind/mujoco#1398]
Hi,

I'm a student and I'm trying to use MuJoCo for Sim2Real of a gripper.

I'm looking for some help with System Identification.

1. You mentioned early the guidelines for system identification will be released later in [Issue](https://github.com/google-deepmind/mujoco/issues/1075#issuecomment-1755996016), I'm wondering where I can find it. Thanks! 

In case it's still not available, it would be also really nice if you could provide some references.

2. My problem is demonstrated below in case you need it. I want to match an XM430 actuator in the simulation based on the parameters from the OP3 (diagram 1). I want to align this with my physical actuator with slower acceleration. (diagram 2) Do you know what are the key parameters I should adjust? And while I'm adjusting parameters, is there a recommended method that could adjust the parameters iteratively instead of manually?

![XM430_simulation](https://github.com/google-deepmind/mujoco/assets/75078611/c3065fa1-4aee-4db4-a2a8-13a4a21b7f7b)

![Model_Dynamics_20240202_221037](https://github.com/google-deepmind/mujoco/assets/75078611/4ea3e821-5b0a-4fc1-84b6-ed8930119cad)

**Comment by yuvaltassa** [comment]
Unfortunately sysID best-practices are not yet published, but @kevinzakka and I are finally making some progress. Thank you for motivating us to finish this 🙂.

TL;DR you want to make a box-constrained least squares problem that minimizes (real_data - simulated_data) as a function of the model parameters, subject to physicality constraints (e.g. no negative damping etc).

You probably want to adjust damping, frictionloss, armature and gain.

Closing for now, hopefully will have updates soon.

**Comment by ykim104** [comment]
Do you have updates on this? Thanks in advance for your reply! :) @yuvaltassa

**Comment by yuvaltassa** [comment]
Unfortunately no 😞 
It's still on our to-do list!
This will happen.

**Comment by Stocko-2073** [comment]
i'm hoping this is the year 🤞

**Comment by kevinzakka** [comment]
👀

**Comment by yuvaltassa** [comment]
Released! See 3.5 changelog 🙂
